modules/apps/content-dashboard/content-dashboard-api/src/main/java/com/liferay/content/dashboard/item/ContentDashboardItemVersion.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.content.dashboard.item;
+
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.json.JSONUtil;
+
+import java.util.Date;
+
+/**
+ * @author Jürgen Kappler
+ */
+public class ContentDashboardItemVersion {
+
+	public ContentDashboardItemVersion(
+		String label, String style, String version, String changeLog,
```

Comment:
Do you think we should sort the parameters in alphabetical order?

Reply:
I'll change this to avoid issues @cgoncas, thanks!